src/PayID/pay-id-client.ts

```diff
@@ -43,6 +52,140 @@ export default class PayIDClient {
    * @param useHttps Whether to use HTTPS when making PayID requests. Most users should set this to 'true' to avoid
    *                 Man-in-the-Middle attacks. Exposed as an option for testing purposes. Defaults to true.
    */
+  constructor(network: string, useHttps: boolean = true) {
+    this.wrappedPayIdClient = new PayIdClient(network, useHttps)
+  }
+
+  /**
+   * Retrieve the address associated with a PayID.
+   *
+   * @param payID The payID to resolve for an address.
+   * @returns An address representing the given PayID.
+   */
+  async addressForPayID(payID: string): Promise<CryptoAddressDetails> {
+    return this.wrappedPayIdClient.addressForPayId(payID)
+  }
+
+  /**
+   * Generate a new invoice with compliance requests.
+   *
+   * @param payID The Pay ID to request an invoice for.
+   * @param nonce A randomly selected nonce that is unique to this invoice request.
+   */
+  async getInvoice(
+    payID: string,
+    nonce: string,
+  ): Promise<SignatureWrapperInvoice> {
+    return this.wrappedPayIdClient.getInvoice(payID, nonce)
+  }
+
+  /**
+   * Post an invoice with compliance data.
+   *
+   * @param payID The Pay ID compliance data is associated with.
+   * @param publicKeyType The type of public key.
+   * @param publicKeyData An array of public keys which lead back to the root trust certificate.
+   * @param publicKey The public key.
+   * @param signature The signature of the operation.
+   * @param originatorUserLegalName The legal name of the originator.
+   * @param originatorAccountID The account ID of the originator.
+   * @param originatorUserPhysicalAddress The physical address of the originator.
+   * @param originatorInstitutionName The institution name of the originator.
+   * @param valueAmount The value being transferred.
+   * @param valueScale The scale of the value.
```

Comment:
The scale of the value. - Is it clear what this is, or what the value means?

Reply:
Ah, I think I should say `valueAmount`? 

Happy to hear other revisions to this documentation to. 

The idea is that $1 USD at scale 2 is 100. Or $1.23USD is 123 at scale 2. Scale is the number of significant decimal places? 

This is terminology that is borrowed from ILP. All their APIs take a value and a scale. Maybe they've found a better way to describe it?

Reply:
I think XRPL uses the same notion of an amount and scale.